ui/src/main/resources/edu/wpi/grip/ui/templates/java/operations/Distance_Transform.vm

```diff
@@ -0,0 +1,74 @@
+	/**
+	 * An indication of which type of transform.
+	 */
+	enum Distance_TransformType{
```

Comment:
Very weird naming scheme. Use `CamelCase`. This also isn't needed because it just maps to a few constants defined by OpenCV.